contracts/MerkleLib.sol

```diff
@@ -105,4 +106,23 @@ library MerkleLib {
         uint256 claimedBitIndex = index % 256;
         claimedBitMap[claimedWordIndex] = claimedBitMap[claimedWordIndex] | (1 << claimedBitIndex);
     }
+
+    /**
+     * @notice Tests whether a claim is contained within a 1D claimedBitMap mapping.
+     * @param claimedBitMap a simple uint256 value, encoding a 1D bitmap.
+     * @param index the index to check in the bitmap.
+     \* @return bool indicating if the index within the claimedBitMap has been marked as claimed.
+     */
+    function isClaimed1D(uint256 claimedBitMap, uint256 index) public pure returns (bool) {
+        return claimedBitMap & (1 << index) > 0;
```

Comment:
nit: can you explain the differences in logic from the uniswap one?

I think their version of this code is:
```
        uint256 mask = (1 << index);
        return claimedBitMap & mask == mask;
```

Not trying to be nit-picky, this logic is just very dense, and I want to make sure we're super confident in the correctness.

Reply:
going to add unit tests now on this and will verify the behaviour. can test out this other code example as well (note I created my implementation from modifying the uniswap code to be 1 dimensional :) )

Reply:
both work equivalently. I reverted to your recommendation as it's easier to read.